### PR TITLE
Update readme things

### PR DESCRIPTION
**Pull request by olstenlarck**
- Updates a bit of the wording and some typos.
- Extends the Developing section it more

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "license": "ISC",
   "scripts": {
     "lint": "standard",
+    "dev": "nodemon node service.js",
```
Comment (amio) [inline package.json]: Why is it `nodemon node service.js` other than `nodemon service.js`?

**Comment by olstenlarck** [comment]
Done :D

**Comment by amio** [comment]
Thanks @olstenlarck!

File: package.json (hunk 1 above)
Comment (olstenlarck) [reply package.json]: oh.. yea, i forgot about that... :D

**Pull request merged**